moped-database/migrations/1608323427908_alter_table_public_moped_project_add_column_project_extent/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."moped_project" ADD COLUMN "project_extent_ids" json NULL;
```

Comment:
Use jsonb instead of regular json, performance is better and we can index certain fields. Also, we get to use really cool database operators.

https://hasura.io/blog/postgres-json-and-jsonb-type-support-on-graphql-41f586e47536/

Reply:
Notes on differences:
https://www.compose.com/articles/faster-operations-with-the-jsonb-data-type-in-postgresql/#:~:text=What's%20jsonb,string%2C%20but%20as%20binary%20code.

Reply:
Thanks for sharing these articles. They fits nicely in with those DB videos that I watched last week. 📚